include/geom/cell_hex8.h

```diff
@@ -184,6 +184,17 @@ class Hex8 final : public Hex
    */
   static const unsigned int edge_sides_map[num_edges][2];
 
+  /**
+   * Class static helper function that computes the centroid of a
+   * hexahedral region from a set of input points which are assumed to
+   * be in counter-clockwise order. Hex8::true_centroid() and
```

Comment:
What does it mean for points in 3D to be in counter-clockwise order?  If you can look along a "z" axis with respect to which the points' angle theta is increasing counter-clockwise, can't you just look along the same axis in the opposite direction and see the angle going clockwise?

Reply:
Uh, yeah this comment is poorly-worded. The function works on a set of points that are numbered "like a Hex8", including if you e.g. collapse a face down to a single point (Pyramid case) or an edge (Prism6 case). I'm happy to change the comment to whatever.